exasol/slc/internal/tasks/test/test_container_file_info.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class TestContainerFileInfo:
```

Comment:
TestContainer is ambiguous, either SLCFileInfo or ContainerFileUnderTestInfo,

Reply:
Fixed